### PR TITLE
Diff with HEAD and master to get the correct result

### PR DESCRIPTION
**Pull request by chuanjin**
In case there are multiple commits, it is safe to diff HEAD with master

### CODE REVIEW

**Comment by chuanjin** [comment]
Since there might be multiple commits within one pull request, if we check against the last commit, there might be no changes. Imaging there are two commits first one updated manifest, but second did sth else, then we might miss to catch the diff. Check against master will be sure to catch the changes.

**Comment by tobsan** [comment]
I understand, but what I meant was that perhaps "pyro" or "rocko" is the branch one wants to check against. Could we make it optional with a default setting of master?

File: yocto/shortlog.sh [hunk 1]
```diff
@@ -26,14 +26,16 @@
 MANIFEST=$1
 YOCTO_BUILD_DIR="$2"
 SOURCE_CODE_ROOT="$3"
+BRANCH="$4"
+BRANCH=${BRANCH:-master}
```
Comment (tobsan) [inline yocto/shortlog.sh]: Can you change this to `${4:-master}` please?

Comment (chuanjin) [reply yocto/shortlog.sh]: Sure, Done!

**Pull request merged**